spec/factories.rb

```diff
@@ -6,12 +6,15 @@
 
   factory :expense do
     sequence(:name) { |n| "name#{n}" }
-    frequency "Weekly"
+    frequency "weekly"
+    due_date "01/02/2017"
+    bill_estimate 600
```

Comment:
Only name and frequency are required for a valid expense. So we can remove lines 10-11

Reply:
Done in sha: 9092c10